commands/config/set.ts

```diff
@@ -1,5 +1,5 @@
 import { ArgumentsCamelCase, Argv } from 'yargs';
```

Comment:
Some updates to the TS patterns in here, but the work in this file adds support for `hs config set --allow-auto-updates`